include/miopen/miopen.h

```diff
@@ -698,7 +708,19 @@ MIOPEN_EXPORT miopenStatus_t miopenSetTensorDescriptor(miopenTensorDescriptor_t
                                                        const int* dimsA,
                                                        const int* stridesA);
 
-/*! @brief Get size of N-dimensional tensor
+/*! @brief Set the tensor cast type
+ *
+ *  For tensors where the cast_type attribute is set, the tensor elements would be cast to the
```

Comment:
[Recommendation] @JehandadKhan Maybe the word "conversion" is a better fit? The "cast" is something from the C/C++ programmer's and magican's vocabulary (which is not surprising because they are almost the same except wands ;)

Reply:
Fixed!